contracts/Fund.sol

```diff
@@ -143,6 +148,29 @@ contract Fund is Administratable {
     emit GrantCreated(grantId, newGrant);
     pendingGrants[grantId] = newGrant;
     grantIds.push(grantId);
+  } 
+
+  function updateGrant(
```

Comment:
Is `updateGrant` necessary? Took me 3 minutes to write; if yes, will write tests; if no, will delete.

Reply:
^see previous conversation

Reply:
In the words of @rheeger: ^see previous conversation 😝

(@CantelopePeel or anyone else, feel free to unresolve as always. This PR is fairly big so to help push it towards mergeable state I'm resolving discussions that don't seem contentious)